.travis.yml

```diff
@@ -4,13 +4,12 @@ cache:
   directories:
     - $HOME/.composer/cache
     - vendor
+    - $HOME/cphalcon
 php:
 - 5.3
 - 5.4
 - 5.5
 - 5.6
-- 7
```

Comment:
My bad... I'm kinda new to Travis and I've really only used a couple times, let alone with Phalcon. I'll fix it up right away.